dist/jdataview.js

```diff
@@ -1,23 +1,23 @@
-(function (factory) {
-	var global = this;
-
-	if (NODE || typeof exports === 'object') {
-		module.exports = factory(global);
-	} else
-	if (BROWSER) {
-		if (typeof define === 'function' && define.amd) {
-			define([], function () {
-				return factory(global);
-			});
-		}
-		else {
-			global.jDataView = factory(global);
-		}
-	}
-}(function (global) {
-
-'use strict';
-
+(function (factory) {
```

Comment:
Ah, nm, you're right. It's different newlines. I'm on a Mac, perhaps that's
why?

On Thu, Sep 10, 2015 at 2:10 PM Ingvar Stepanyan <notifications@github.com>
wrote:

> In dist/jdataview.js
> <https://github.com/jDataView/jDataView/pull/72#discussion_r39193230>:
>
> > -	} else
> > -	if (BROWSER) {
> > -		if (typeof define === 'function' && define.amd) {
> > -			define([], function () {
> > -				return factory(global);
> > -			});
> > -		}
> > -		else {
> > -			global.jDataView = factory(global);
> > -		}
> > -	}
> > -}(function (global) {
> > -
> > -'use strict';
> > -
> > +(function (factory) {
>
> That's weird, Github wouldn't show difference if it wouldn't be there.
> Maybe different newlines and local git ignores them? If you don't find
> the reason, I can try on my own a bit later.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jDataView/jDataView/pull/72/files#r39193230>.
>